tests/openapi/run.sh

```diff
@@ -42,6 +42,54 @@ function test_source() {
 	echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>TEST OPENAPI: SOURCE SUCCESS"
 }
 
+function test_relay() {
+	echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>START TEST OPENAPI: RELAY"
+	prepare_database
+	# create source succesfully
+	openapi_source_check "create_source1_success"
+
+	# start relay failed
+	openapi_source_check "start_relay_failed" "mysql-01" "no-worker"
```

Comment:
Why will we fail here 🤔

Reply:
because this cmd try to start relay on not exists worker